include/SSVOpenHexagon/Core/HexagonDialogBox.hpp

```diff
@@ -17,31 +17,67 @@ namespace hg
 class HGAssets;
 class StyleData;
 
+enum DBoxDraw
+{
+    topLeft = 0,
+    center,
+    centerUpperHalf
+};
```

Comment:
Please change this to an `enum class` so that the enumerator names do not leak in the surrounding namespace. This will require you explicitly saying `DBoxDraw::topLeft` rather than just `topLeft` on the usage site, which is a good thing for readability and avoiding name collisions. 

```suggestion
enum class DBoxDraw
{
    topLeft = 0,
    center,
    centerUpperHalf
};
```